floodviz/maputils.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+# function to generate a dictionary of site data after querying NWIS
+def site_dict(site_list, url_prefix):
```

Comment:
Need to add comments describing the function, the inputs, and what it returns. Generally we use sphinx standards. You can go here for an example: https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions